scripts/i18n.js

```diff
@@ -39,7 +39,7 @@ $(() => {
   *                                          the captions for "key1" and "key2", respectively.
   */
 
-  i18n.on('initialized languageChanged', () => {
+  i18n.on('languageChanged', () => {
```

Comment:
Might wrap this inside a [loaded event](http://i18next.com/docs/api/#on-loaded) to prevent updating DOM when opening offline.